terasoluna-gfw-common-libraries/terasoluna-gfw-web/src/main/java/org/terasoluna/gfw/web/el/Functions.java

```diff
@@ -175,14 +175,7 @@ public static String u(String value) {
         if (value == null || value.isEmpty()) {
             return "";
         }
-        try {
-            return UriUtils.encodeQueryParam(value, "UTF-8");
-        } catch (UnsupportedEncodingException ignored) {
```

Comment:
- [ ] Oh... Please remove `import java.io.UnsupportedEncodingException;`.

Reply:
It has been deleted.

Reply:
OK